docs/meaning_for_configurations_zh.md

```diff
@@ -1,3 +1,245 @@
+        # Frequency for testing. For example, `test_freq=3` means the performance is tested every 3 global epochs.
+        test_freq=3,
+        # What is the logging directory of this experiment.
+        # If the directory does not exist, it will be created automatically.
+        # If the directory already exists, some parts might be over-written, which should be carefully inspected.
+        logging_path='./logging/default_experiment',
+        # The saved model checkpoint to start from. If it is set to ``None``, the training process
+        # will start from scratch.
+        resume_path=None,
+        # Whether to print config is the command line.
+        print_config=False,
+    ),
+)
+```
+
+## 其他教程
+
+### learn.scheduler
+
+此键用于控制学习率调度器。除了 "fix" 外，`learn.scheduler` 键还具有多种类型。例如，我们可以使用余弦函数定义一个调度器：
+
+```python
+scheduler=dict(
+    # Cosine learning rate scheduler.
+    name='cos'
+    # The `min_lr` is the lower bound of learning rate.
+    min_lr=1e-6,
+    # This means that the learning rate will decay progressively for 30 global epochs and finally reaches `min_lr`.
+    decay_round=30
+)
+```
+
+对于其他类型的调度器，请参考 `fling.utils.LRScheduler`。
+
+### learn.finetune_parameters
+
+这个键用于控制哪些参数应该进行微调。除了 "all" 之外，`learn.finetune_parameters` 键还具有其他值。例如，如果您只想微调参数名称中**包含**关键词 "fc" 或 "bn" 的参数，可以这样写：
+
+```python
+finetune_parameters=dict(
+    # This means to only fine-tune parameters whose names have keywords listed in `keywords`
+    name='contain',
+    keywords=['fc', 'bn']
+)
+```
+
+相反，如果您想微调参数名称**不包含**关键字 "fc" 或 "bn" 的参数，可以这样写：
+
+```python
+finetune_parameters=dict(
+    # This means to only fine-tune parameters except those whose names have keywords listed in `keywords`
+    name='except',
+    keywords=['fc', 'bn']
+)
+```
+
+`group.aggregation_parameters` 控制在联邦学习框架下应该聚合哪些参数，对于它的设置和上面所述的 `learn.finetune_parameters` 是类似的。
+
+### launcher
+
+- 除了 "serial" 外，`launcher.name` 键还有其他用途。在除了 Linux 以外的操作系统上，默认配置 "serial" 意味着每个客户端上的所有操作都是串行执行的，这可能不够高效。我们可以使用多进程方法加速这个过程。
+- 在 Linux 系统上，默认的 `launcher.name` 是 "multiprocessing"，进程数为 `num_proc=2`：
+
+```python
+launcher=dict(
+    name='multiprocessing',
+    num_proc=2
+)
+```
+
+一些需注意的重要事项：
+
+1) 对于 Windows 用户，由于 multiprocessing + PyTorch 在 Windows 上的支持并不好，这种多进程模式可能不稳定，会导致意料之外的错误。如果您遇到与 Windows 上的多进程相关的 bug，请考虑改用串行 launcher。
+2) 由于 PyTorch 2.0 的新功能，编译后的模型目前暂不支持多进程执行。在我们的设计中，如果您启用了多进程模式，PyTorch 的编译将被禁用。这个 bug 可能会在此 pr 完成后由 PyTorch 解决：https://github.com/pytorch/pytorch/pull/101651
+3) `num_proc` 指的是程序中使用的进程数。这个值的最优值可能会受到您执行的任务、所使用的硬件、神经网络、数据等的影响...... 您应该在您的系统上仔细调整这个值，以寻找最佳的选择。以下是在具有 4 核 CPU 的 A100 上，batch_size=32 的简单基准测试：
+
+| num_proc          | 1     | 2     | 3     | 4     | 8     |
+| ----------------- | ----- | ----- | ----- | ----- | ----- |
+| MNIST + CNN       | 21.7s | 14.9s | 15.3s | 16.5s | 25.2s |
+| CIFAR10 + ResNet8 | 45.2s | 32.1s | 30.1s | 32.5s | 40.4s |
+
+### data.transforms
+
+- `include_default` 键用于指定是否包括默认的数据增强方法。例如，如果您正在使用 CIFAR100 数据集并设置 `include_default=True` ，那么您使用的实际数据变换将是：
```

Comment:
Add descriptions about default augmentaton and examples, such as: where the default augmentation of CIFAR100 is defined, what is the aug